pkg/cvo/sync_worker.go

```diff
@@ -184,15 +184,18 @@ type SyncWorker struct {
 	// of the form exclude.release.openshift.io/<identifier>=true
 	exclude string
 
-	// includeTechPreview is set to true when the CVO should create resources with the `release.openshift.io/feature-gate=TechPreviewNoUpgrade`
-	includeTechPreview bool
+	// requiredFeatureSet is set to the value of Feature.config.openshift.io|spec.featureSet.
+	// The CVO should create resources with the `annotations[release.openshift.io/feature-set]` unset or if
+	// the annotation is set, it must contain the requiredFeatureSet.
+	// The library called by the CVO translates "" into "Default" to ease usage.
```

Comment:
We might not need to get into this level of detail about what's going on inside the vendored library.  Maybe something more generic like:

> requiredFeatureSet is set to the value of Feature.config.openshift.io|spec.featureSet, which contributes to whether or not some manifests are included for reconciliation.